content/docs/components/metadata.md

```diff
@@ -150,31 +148,12 @@ artifact in the **Artifact Store** on the Kubeflow UI.
 
 
 
-## Backend and REST API
+## GRPC backend
 
-The Kubeflow metadata backend uses [ML Metadata
-(MLMD)](https://github.com/google/ml-metadata/blob/master/g3doc/get_started.md) 
-to manage the metadata and relationships. 
+The Kubeflow metadata deploys the [gRPC service](https://github.com/google/ml-metadata/blob/master/ml_metadata/proto/metadata_store_service.proto) of [ML Metadata
+(MLMD)](https://github.com/google/ml-metadata/blob/master/g3doc/get_started.md) to manage the metadata and relationships.
 
-The backend exposes a 
-[REST API](/docs/reference/metadata/v1alpha1/kubeflow-metadata-api-spec/).
-
-You can add your own metadata types so that you can log metadata for custom
-artifacts. To add a custom type, send a REST API request to the
-[`artifact_types` endpoint](/docs/reference/metadata/v1alpha1/kubeflow-metadata-api-spec/#operation--api-v1alpha1-artifact_types-post). 
-
-For example, The following request registers an artifact type with 
-_name_ `myorg/mytype/v1` and three _properties_: 
-
-* `f1` (string)
-* `f2` (integer)
-* `f3` (double)
-
-```
-curl -X POST http://localhost:8080/api/v1alpha1/artifact_types \
-  --header "Content-Type: application/json" -d \
-  '{"name":"myorg/mytype/v1","properties":{"f1":"STRING", "f2":"INT", "f3": "DOUBLE"}}'
-```
+Kubeflow metadata SDK saves and UI retrieves data via the gRPC service. Similarly, you can define your own metadata types to log and view metadata for your custom artifacts. For Python examples, you can check [MLMD Python client](https://pypi.org/project/ml-metadata/) and Kubeflow metadata SDK [source code](https://github.com/kubeflow/metadata/blob/master/sdk/python/kubeflow/metadata/metadata.py). For Go examples, you can check the Watcher [source code](https://github.com/kubeflow/metadata/blob/master/watcher/handlers/metalogger.go).
```

Comment:
Suggested revision for: Kubeflow metadata SDK saves and UI retrieves data via the gRPC service.

The Kubeflow Metadata SDK saves and retrieves data via the gRPC service.

Reply:
metadata SDK -> Metadata SDK

Reply:
What is "the Watcher"? Could this be referred to as the resource_watcher, or some other descriptive name?

Reply:
Done.